### DISCUSSION TITLE
Best pattern for providing analytics to customers / end-users?

### DISCUSSION THREAD

**Discussion by aubreythreadgill** [plausible/analytics#3548] [Anything Else]
I'm using Plausible to provide the end-users of my web app [https://tapistr.ee] with their own analytics. I would greatly appreciate hearing from anyone who has used Plausible for this use case.

**Desired Outcome**
Using X/Twitter as an example, we want to give users Alice (x.com/Alice) and Bob (x.com/Bob) each their own analytics pages, and we don't want them to be able to see one another's analytics.

**Proposed Solution**
1. Add `<Helmet>  <script defer data-domain={`tapistr.ee/${username}`} src="https://plausible.io/js/script.outbound-links.exclusions.js" data-exclude="/i/analytics"></script></Helmet>` to the profile page.
2. In Plausible, add a New Site [https://plausible.io/sites/new] for `x.com/Alice` and  `x.com/Bob`.
3. In Plausible (under 'Visibility'), generate a Shared Link, create an Embed, and add an iframe with the analytics url behind an authorized page.

**Potential Issues**
1. I've implemented this and also have Plausible on my root route. I see slight discrepancies between `/Alice` on my dashboard and `/Alice` on her dashboard. Don't have a theory on why this is the case.
2. I need to review the API documentation, but this may involve me manually creating new sites in the dashboard for new users.
3. The 'Business' pricing plan only allows up to 50 sites, and therefore only 50 users.

Thanks in advance for any suggestions and pointing out any flaws in my thinking!

**Comment by metmarkosaric** [comment]
Hi @aubreythreadgill! The proposed solution is all correct. Some of our biggest subscribers are using Plausible for exactly this use case. 

If you used the same shared link for the embed there should be no differences in stats as it's the same shared link. The slight difference you're seeing is probably because our dashboard is real time so every time you load the dashboard the stats will change if you've had some activity on the site between the reloads. 

If you need more than 50 sites, we can accommodate that with a custom plan. 

Best place for these types of questions or for any other help is our customer support. See https://plausible.io/contact

Thanks!

**Reply by aubreythreadgill** [reply]
Excellent, thank you Marko! I'll reach out
